packages/angular/projects/ui-angular/src/lib/primitives/amplify-form-field/amplify-form-field.component.ts

```diff
@@ -18,10 +18,10 @@ import { StateMachineService } from '../../services/state-machine.service';
  * that does not make any auth-related inference.
  */
 @Component({
-  selector: 'amplify-form-input',
-  templateUrl: './amplify-input.component.html',
+  selector: 'amplify-form-field',
+  templateUrl: './amplify-form-field.component.html',
 })
-export class AmplifyInputComponent implements OnInit {
```

Comment:
This is not really a primitive because it's opinionated to auth usage. I'll move this out of the primitive folder in a later PR.